integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -6280,6 +6310,10 @@ msgstr "Reihenfolge"
 msgid "Embed live content"
 msgstr "Live-Inhalte einbinden"
 
+#: cms/templates/pages/page_form_sidebar/settings_box.html
+msgid "Embed offers"
+msgstr "Eingebettete Angebote"
```

Comment:
```suggestion
msgstr "Angebote einbinden"
```
This could be just personal preference, but I like "einbidne" a bit more because it suggests activity and therefore maybe helps to communicate the call to action a bit better?